<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book A Ticket</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="navigationBG" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#navigationBG {
background-color: #ccdde8;

}

</string>
    </property>
    <widget class="QPushButton" name="logoutButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>550</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/logout.png</normaloff>../img/logout.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="systemLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>15</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#systemLabel {
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: #034488;
}</string>
     </property>
     <property name="text">
      <string>Book A Ticket</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/icon.png</normaloff>../img/icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="rect1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#rect1{
background-color: white;
border: 1px white;
border-radius: 20px;
}</string>
    </property>
    <widget class="QComboBox" name="movieTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#movieTitle{
border: 1px white;
}</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>Movie Title</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="rect2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#rect2{
background-color: white;
border: 1px white;
border-radius: 20px;
}</string>
    </property>
    <widget class="QComboBox" name="cinemaName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#cinemaName{
border: 1px white;
}</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>Cinema</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="rect3" native="true">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>190</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#rect3{
background-color: white;
border: 1px white;
border-radius: 20px;
}</string>
    </property>
    <widget class="QSpinBox" name="noOfTickets">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#noOfTickets{
border: 1px white;
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bookTicketButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#bookTicketButton{
background-color: #ccdde8;
border: 1px white;
border-radius: 20px;
}
#bookTicketButton:hover{
background-color: #ccdde8;
border: 1px black;
border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string>Book Ticket</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>170</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No. of tickets</string>
    </property>
   </widget>
   <widget class="QWidget" name="rect4" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#rect4{
background-color: white;
border: 1px white;
border-radius: 20px;
}</string>
    </property>
    <widget class="QComboBox" name="schedule">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#schedule{
border: 1px white;
}</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Schedule</string>
    </property>
   </widget>
   <zorder>rect4</zorder>
   <zorder>navigationBG</zorder>
   <zorder>rect1</zorder>
   <zorder>rect2</zorder>
   <zorder>rect3</zorder>
   <zorder>bookTicketButton</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
